python-client/giskard/scanner/performance/metrics.py

```diff
@@ -113,6 +114,38 @@ class MeanAbsoluteError(SklearnRegressionScoreMixin, RegressionPerformanceMetric
     _sklearn_metric = "mean_absolute_error"
 
 
+class ProbaMAE(PerformanceMetric):
+    name = "probamae"
+    greater_is_better = False
+
+    def __call__(self, model: BaseModel, dataset: Dataset) -> float:
+        if not model.is_classification:
+            raise ValueError(f"Metric '{self.name}' is only defined for classification models.")
+
+        ocd = OverconfidenceDetector(model, dataset)
+
+        return self._calculate_metric(ocd)
+
+    def _calculate_metric(self,ocd) -> float:
+        return ocd.get_proba_rmse()
+
+class Borderline(PerformanceMetric):
+
+    name = "borderline"
+    greater_is_better = True
+
+    def __call__(self, model: BaseModel, dataset: Dataset) -> float:
+        if not model.is_classification:
+            raise ValueError(f"Metric '{self.name}' is only defined for classification models.")
+
+        bld = BorderlineDetector(model, dataset)
+
+        return self._calculate_metric(bld)
+
+    def _calculate_metric(self, bld) -> float:
+        return bld.get_proba_rmse()
+
+
```

Comment:
Looks too specific to define a performance metric. This should go somewhere else.